cirq-core/cirq/devices/grid_device_metadata.py

```diff
@@ -129,8 +161,9 @@ def _json_dict_(self):
             'qubit_pairs': sorted(list(self._qubit_pairs)),
             'gateset': self._gateset,
             'gate_durations': duration_payload,
+            'isolated_qubits': sorted(list(self._isolated_qubits)),
         }
 
     @classmethod
-    def _from_json_dict_(cls, qubit_pairs, gateset, gate_durations, **kwargs):
-        return cls(qubit_pairs, gateset, dict(gate_durations))
+    def _from_json_dict_(cls, qubit_pairs, gateset, gate_durations, isolated_qubits, **kwargs):
+        return cls(qubit_pairs, gateset, dict(gate_durations), isolated_qubits)
```

Comment:
not backwards compatible

Reply:
This functionality hasn't been released yet. I don't think we need to worry just yet about backwards compatibility.

Reply:
kk